dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -256,8 +256,8 @@ def load_section
       if section.user == current_user
         @section = section
       end
-    elsif current_user.try(:sections) && current_user.sections.count == 1
-      @section = current_user.sections.first
+    elsif current_user.try(:non_deleted_sections) && current_user.non_deleted_sections.count == 1
+      @section = current_user.non_deleted_sections.first
```

Comment:
Any reason this can't be simplified to
```
elsif current_user && current_user.non_deleted_sections.count == 1
  @section = current_user.non_deleted_sections.first
```
Presumably, the need for the `try` is because `current_user` may be `nil`?